test/data/data_source/genesyscloud_journey_action_map/find_by_name/01_find_by_name.tf

```diff
@@ -19,7 +19,6 @@ resource "genesyscloud_journey_action_map" "terraform_test_-TEST-CASE-" {
 resource "genesyscloud_journey_segment" "terraform_test_-TEST-CASE-_action_map_dependency" {
   display_name            = "terraform_test_-TEST-CASE-_action_map_dependency"
   color                   = "#008000"
-  scope                   = "Session"
```

Comment:
Most of the files below this point just contain the removal of scope from test files.

A few of them I deleted the whole file altogether, because they weren't doing anything meaningful, given all the removed fields.

For example, there was a test step 04_update_remove_assignment_expirition_days.tf, but expiration days were removed in the last PR, so no need for it anymore.